test/e2e/specs/all-features-confirmation/recordedit/add.spec.js

```diff
@@ -212,10 +221,43 @@ describe('Record Add', function() {
                 }
             });
         });
-
+        describe('Markdown Editor Help button is clicked, ', function() {
```

Comment:
I put this inside the outer `describe` just to avoid opening browser for this one case.